### DISCUSSION TITLE
Frontend clients to connect and query comunica sparql endpoints

### DISCUSSION THREAD

**Discussion by allforabit** [comunica/comunica#787] [Q&A] [answered]
Hi there, I'm just wondering if anyone knows of a graphical client of some sort to run queries against and to explore a dataset? I'm currently using node quadstore and this uses comunica as the sparql engine. I currently do a lot of console logging to see what data I have in there but would like to have something a bit more visual and easy to use. Something like phpMyAdmin for sql. Any recommendations would be greatly appreciated! Thanks :-)

**Comment by rubensworks** [comment, marked as answer]
I tend to just do some SPARQL queries on http://query.linkeddatafragments.org/ to explore a dataset :-)

But I guess you're looking for some kind of graph view that can be supplied with any given dataset?
If so, this is not the immediate aim of Comunica, but I would definitely also be interested in this!

**Reply by allforabit** [reply]
Ah I hadn't noticed that this was an open source project https://github.com/comunica/jQuery-Widget.js Thanks I think that'll do me for now. Will report back if I find anything else. Yes, it's really begging for a graph explorer of some kind maybe using something like this: https://js.cytoscape.org/. Might even have a play around with that :-)
